src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -268,6 +271,12 @@ public async Task InitializeAsync()
 
         private async Task LogInitializationAsync()
         {
+            // If the host id is explicitly set, emit a warning that this could cause issues and shouldn't be done
+            if (_configuration[ConfigurationSectionNames.HostIdPath] != null)
+            {
+                _logger.LogWarning("Host id explicitly set in the host.json. It is recommended that you remove the \"id\" property in your host.json.");
```

Comment:
This isn't necessarily coming from host.json -- so this warning should be cleaned up.

Reply:
Good point. Meant to modify this and didn't get back to it.